static/fonts/flaticon/flaticon.html

```diff
@@ -0,0 +1,575 @@
+<!DOCTYPE html>
```

Comment:
Do we need this file?

Reply:
Yes, we need this file for development purposes. This is a demo file. How will you recognize which class use for a specific icon?

Reply:
Ok, thanks for clarification :)